xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/Ratings.xml

```diff
@@ -394,9 +394,10 @@ position: relative;
     <className>XWiki.UIExtensionClass</className>
     <guid>d2420ff0-a5b9-4fae-a798-fc623aac7a6b</guid>
     <property>
-      <content>{{include reference='XWiki.RatingsMacros'/}}
-
-{{velocity}}
+      <content>{{velocity}}
+#if (!$isGuest)
```

Comment:
Hi Victor! Cool to see you active.

I'm not sure we want guest users to be able to rate something since we want users to rate only once AFAIR, right?

So we do need to check if the user is not guest. We also need to check if he has permissions to rate something and I don't know what's involved there (view access on the XClass but probably more rights to add the rating page).

Thanks